### ISSUE TITLE
Reshuffle/Rebalance Modern Upgrades

### ISSUE THREAD

**Issue by weylint** [D3nnis3n/White-Tiger-Feedback#55]
We had issues getting the MU-Chain going 2 cycles in a row and I do assume it's because the first few steps have too many complexities. Without changing the overall ressource usage, I'd suggest sorting them by complexity with a slight reshuffle.

Since #5 got implemented, nowadays the MU Chain is as follows (with dependancies):

Industry MU1: Steel Plates (Industry), Steel Gear (Industry+Oil)
Composites MU2: Rivets (Adv. Smelting), Steel Plates (Industry), Composite Lumber (Composite)
Adv. Masonry MU3: Copper Wiring (Mechanics), Gold Wiring (Mechanics), Ashlar (Adv. Masonry, Adv. Smelting)
Electronics MU4: Basic Circuits (Mechanic, Electronic, Oil), Plastic (Oil), Epoxy (Oil)

Industry MU1: Steel Plates, **Rivets**
Adv. Masonry MU2: As is
Composites MU3: **Steel Gear**, Steel Plates, Composite Lumber
Electronics MU4: As is.

Removing Oil from MU1s makes them less complex. So the MU Chain gets going easier. Instead add Steel Pipes or Rivets to MU1 and give the Steel Gears to Composites.
Switching MU2/MU3 also feels natural, because Adv. Masonry MU isn't easy to make, but has less dependencies on advanced jobs, just high ressource requirements in general.

At this point oil comes into play and can start out with *some* kind of MU.
MU3 is where oil starts becoming a requirement.
MU4 is the high tech stuff that has lots of different requirements.

**Comment by DanielKispert** [comment]
would make the modern upgrade chain weak to oil races like described in #32

**Comment by weylint** [comment]
Yeah, that's an issue. So without violating that, it could at most be exchanging Rivets with Steel Gear requirements? Would keep races in check in general, just not for MU1s, which aren't terribly high of a bonus. Might help with kickstarting MU-Chain.

**Comment by Tralolol** [comment]
Make all upgrades able to be crafted by any skill and then find a balance of resources per each skill. Why is monopoly allowed in a free market, what the actual hell?
